qiskit_ibm_runtime/ibm_sampler.py

```diff
@@ -0,0 +1,135 @@
+    primitive. It returns a :class:`qiskit_ibm_runtime.sessions.SamplerSession` instance.
+
+    * circuits: list of (parameterized) quantum circuits
+        (a list of :class:`~qiskit.circuit.QuantumCircuit`))
+
+    * parameters: a list of parameters of the quantum circuits.
+        (:class:`~qiskit.circuit.parametertable.ParameterView` or
+        a list of :class:`~qiskit.circuit.Parameter`) specifying the order
+        in which parameter values will be bound.
+
+    The :class:`qiskit_ibm_runtime.sessions.SamplerSession` instance can be called repeatedly
+    with the following parameters to calculate probabilites or quasi-probabilities.
+
+    * circuit_indices: A list of circuit indices.
+
+    * parameter_values: Concrete parameters to be bound.
+
+    All the above lists should be of the same length.
+
+    Example::
+
+        from qiskit import QuantumCircuit
+        from qiskit.circuit.library import RealAmplitudes
+
+        from qiskit_ibm_runtime import IBMRuntimeService, IBMSampler
+
+        service = IBMRuntimeService(auth="cloud")
+        sampler_factory = IBMSampler(service=service, backend="ibmq_qasm_simulator")
+
+        bell = QuantumCircuit(2)
+        bell.h(0)
+        bell.cx(0, 1)
+        bell.measure_all()
+
+        # executes a Bell circuit
+        with sampler_factory(circuits=[bell], parameters=[[]]) as sampler:
+            result = sampler(circuit_indices=[0], parameter_values=[[]])
+            print([q.binary_probabilities() for q in result.quasi_dists])
+
+        # executes three Bell circuits
+        with sampler_factory([bell]*3, [[]]) as sampler:
+            result = sampler(circuit_indices=[0, 1, 2], parameter_values=[[]]*3)
+            print([q.binary_probabilities() for q in result.quasi_dists])
+
+        # parametrized circuit
+        pqc = RealAmplitudes(num_qubits=2, reps=2)
+        pqc.measure_all()
+        pqc2 = RealAmplitudes(num_qubits=2, reps=3)
+        pqc2.measure_all()
+
+        theta1 = [0, 1, 1, 2, 3, 5]
+        theta2 = [1, 2, 3, 4, 5, 6]
+        theta3 = [0, 1, 2, 3, 4, 5, 6, 7]
+
+        with sampler_factory(circuits=[pqc, pqc2], parameters=[pqc.parameters, pqc2.parameters])
+            as sampler:
+            result = sampler(circuit_indices=[0, 0, 1], parameter_values=[theta1, theta2, theta3])
+            # result of pqc(theta1)
+            print([q.binary_probabilities() for q in result.quasi_dists[0]])
+            # result of pqc(theta2)
+            print([q.binary_probabilities() for q in result.quasi_dists[1]])
+            # result of pqc2(theta3)
+            print([q.binary_probabilities() for q in result.quasi_dists[2]])
+    """
+
+    def __call__(  # type: ignore[override]
+        self,
+        circuits: Iterable[QuantumCircuit],
+        parameters: Optional[Iterable[Iterable[Parameter]]] = None,
+        transpile_options: Optional[Dict] = None,
+        skip_transpilation: bool = False,
```

Comment:
i commented on the internal implementation that i think this kwarg should be removed. There is a `bound_pass_manager` kwarg there instead that essentially accomplishes this. (However i don't see `bound_pass_manager` plus a few other kwargs like `mitigator` exposed here in your code).